modules/luci-mod-dashboard/htdocs/luci-static/resources/view/dashboard/include/10_router.js

```diff
@@ -0,0 +1,322 @@
+				this.params.internet.v4.protocol.value=  ifc.getI18n() || E('em', _('Not connected'));
+				this.params.internet.v4.gatewayv4.value =  ifc.getGatewayAddr() || '0.0.0.0';
+				this.params.internet.v4.connected.value = ifc.isUp();
+				this.params.internet.v4.addrsv4.value = ifc.getIPAddrs() || [ '-'];
+				this.params.internet.v4.dnsv4.value = ifc.getDNSAddrs() || [ '-' ];
+			 }
+		}
+	},
+
+	renderInternetBox: function(data) {
+
+		this.params.internet = {
+
+			v4: {
+				title: 'IPv4 Internet',
+
+				connected: {
+					title: _('Connected'),
+					visible: true,
+					value: false
+				},
+
+				uptime: {
+					title: _('Connected since'),
+					visible: true,
+					value: '-'
+				},
+
+				protocol: {
+					title: _('Protocol'),
+					visible: true,
+					value: '-'
+				},
+
+				addrsv4: {
+					title: _('IPv4'),
+					visible: true,
+					value: [ '-' ]
+				},
+
+				gatewayv4: {
+					title: _('GatewayV4'),
+					visible: true,
+					value: '-'
+				},
+
+				dnsv4: {
+					title: _('DNSv4'),
+					visible: true,
+					value: ['-']
+				}
+			},
+
+			v6: {
+				title: 'IPv6 Internet',
+
+				connected: {
+					title: _('Connected'),
+					visible: true,
+					value: false
+				},
+
+				uptime: {
+					title: _('Connected since'),
+					visible: true,
+					value: '-'
+				},
+
+				protocol: {
+					title: _('Protocol'),
+					visible: true,
+					value: ' - '
+				},
+
+				ipprefixv6 : {
+					title: _('IPv6 prefix'),
+					visible: true,
+					value: ' - '
+				},
+
+				addrsv6: {
+					title: _('IPv6'),
+					visible: false,
+					value: [ '-' ]
+				},
+
+				gatewayv6: {
+					title: _('GatewayV6'),
+					visible: true,
+					value: '-'
+				},
+
+				dnsv6: {
+					title: _('DNSv6'),
+					visible: true,
+					value: [ '-' ]
+				}
+			}
+		};
+
+		this.renderUpdateWanData(data[0], false);
+		this.renderUpdateWanData(data[1], true);
+
+		return this.renderHtml(this.params.internet, 'internet');
+	},
+
+	renderRouterBox: function(data) {
+
+		var boardinfo   = data[2],
+			systeminfo  = data[3];
+
+		var datestr = null;
+
+		if (systeminfo.localtime) {
+			var date = new Date(systeminfo.localtime * 1000);
+
+			datestr = '%04d-%02d-%02d %02d:%02d:%02d'.format(
+				date.getUTCFullYear(),
+				date.getUTCMonth() + 1,
+				date.getUTCDate(),
+				date.getUTCHours(),
+				date.getUTCMinutes(),
+				date.getUTCSeconds()
+			);
+		}
+
+		this.params.router = {
+			uptime: {
+				title: _('Uptime'),
+				value: systeminfo.uptime ? '%t'.format(systeminfo.uptime) : null,
+			},
+
+			localtime: {
+				title: _('Local Time'),
+				value: datestr
+			},
+
+			kernel: {
+				title: _('Kernel Version'),
+				value: boardinfo.kernel
+			},
+
+			model: {
+				title: _('Model'),
+				value: boardinfo.model
+			},
+
+			system: {
+				title: _('Architecture'),
+				value: boardinfo.system
+			},
+
+			release: {
+				title: _('Firmware Version'),
+				value: (L.isObject(boardinfo.release) ? boardinfo.release.description : '')
+			}
```

Comment:
The value is badly shortened. Please use `<version>  (<revision>)` instead of allow the description to be longer.
![image](https://user-images.githubusercontent.com/16000931/88321624-38ac7b80-ccbb-11ea-83b5-9c38eca94001.png)

```
root@gw:~# ubus call system board
{
	"kernel": "5.4.52",
	"hostname": "gw",
	"system": "ARMv7 Processor rev 1 (v7l)",
	"model": "Linksys WRT3200ACM",
	"board_name": "linksys,wrt3200acm",
	"release": {
		"distribution": "OpenWrt",
		"version": "SNAPSHOT",
		"revision": "r13881+3-bae4204e34",
		"target": "mvebu/cortexa9",
		"description": "OpenWrt SNAPSHOT r13881+3-bae4204e34"
	}
}
```

Reply:
Okay, I will update the code later.